Client/InfluxDBClient.cs

```diff
@@ -233,7 +233,96 @@ public class InfluxDBClient : AbstractRestClient, IInfluxDBClient
+        /// </summary>
+        public InfluxDBClient() :
+            this(InfluxDBClientOptions.Builder
+                .CreateNew()
+                .LoadConfig()
+                .Build())
+        {
+        }
+
+        /// <summary>
+        /// Create a instance of the InfluxDB 2.x client. The url could be a connection string with various configurations.
+        /// <para>
+        /// e.g.: "http://localhost:8086?timeout=5000&amp;logLevel=BASIC
+        /// </para>
+        /// </summary>
+        /// <param name="connectionString">connection string with various configurations</param>
+        public InfluxDBClient(string connectionString) :
+            this(InfluxDBClientOptions.Builder
+                .CreateNew()
+                .ConnectionString(connectionString)
+                .Build())
+        {
+        }
+
+        /// <summary>
+        /// Create a instance of the InfluxDB 2.x client.
+        /// </summary>
+        /// <param name="url">the url to connect to the InfluxDB 2.x</param>
+        /// <param name="username">the username to use in the basic auth</param>
+        /// <param name="password">the password to use in the basic auth</param>
+        public InfluxDBClient(string url, string username, char[] password) :
+            this(new InfluxDBClientOptions(url)
+            {
+                Username = username,
+                Password = password
+            })
+        {
+        }
+
+        /// <summary>
+        /// Create a instance of the InfluxDB 2.x client.
+        /// </summary>
+        /// <param name="url">the url to connect to the InfluxDB 2.x</param>
+        /// <param name="token">the token to use for the authorization</param>
+        public InfluxDBClient(string url, char[] token) :
+            this(new InfluxDBClientOptions(url)
+            {
+                Token = token
+            })
+        {
+        }
+
+        /// <summary>
+        /// Create a instance of the InfluxDB 2.x client.
+        /// </summary>
+        /// <param name="url">the url to connect to the InfluxDB 2.x</param>
+        /// <param name="token">the token to use for the authorization</param>
+        public InfluxDBClient(string url, string token) :
+            this(new InfluxDBClientOptions(url)
+            {
+                Token = token.ToCharArray()
+            })
+        {
+        }
+
+        /// <summary>
+        /// Create a instance of the InfluxDB 2.x client to connect into InfluxDB 1.8.
+        /// </summary>
+        /// <param name="url">the url to connect to the InfluxDB 1.8</param>
+        /// <param name="username">authorization username</param>
+        /// <param name="password">authorization password</param>
+        /// <param name="database">database name</param>
+        /// <param name="retentionPolicy">retention policy</param>
+        public InfluxDBClient(string url, string username, char[] password, string database, string retentionPolicy) :
```

Comment:
Please use `string` instead `char[]` for password:

```suggestion
        public InfluxDBClient(string url, string username, string password, string database, string retentionPolicy) :
```